slic3r/export2slicer.FCMacro

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+# Opens current visible objects in Slic3r. One can use different slicing software
+
+__Title__ = "Open in Slic3r"
+__Author__ = "Damian Łoziński"
+__Version__ = "0.1"
+__Date__ = "2019-03-06"
+__Comment__ = "Exports current visible objects to amf files and opens them in Slic3r"
+__Web__ = ""
+__Wiki__ = "http://www.freecadweb.org/wiki/index.php?title=Open in Slic3r"
+__Icon__ = "/usr/lib/freecad/Mod/plugins/icons/Slic3r.png"
+__Help__ = "Run this macro to open visible objects in slicer software. You may need to change slicer path"
+__Status__ = ""
+__Requires__ = "freecad 0.17"
+__Communication__ = "https://github.com/dlozinski"
+__Files__ = "Slic3r.png"
+
+import Mesh
+import MeshPart
+import os
+import subprocess
+
+# Slicing program executable. One can use different slicing software here and provide proper slicer executable location and custom flags
+SLICER_CMD = ["/Applications/Slic3r.app/Contents/MacOS/Slic3r", "--gui"]
+
+# You can format change to stl, but then all of the objects will be marged into single geometry
+OUTPUT_FORMAT = "amf"
+
+# Angular deflection in radians. Lower value = better quality of curved shapes and bigger file size
+ANGULAR_DEFLECTION = 0.0872665
```

Comment:
Rather use `g_angular_deflection` to indicate that a variable is global rather that `ANGULAR_DEFLECTION` which I find inelegant (same for other global variables).

Reply:
done

Reply:
I'm sorry but I learnt inbetween that your former style was the correct one according to PEP-8.